feature-list-streams/src/main/java/com/codandotv/streamplayerapp/feature_list_streams/list/di/ListStreamModule.kt

```diff
@@ -34,7 +35,8 @@ object ListStreamModule {
 
         factory<ListStreamRepository> {
             ListStreamRepositoryImpl(
-                service = get()
+                service = get(),
+                dispatcher = get(named("DispatcherIO"))
```

Comment:
se vc olhar no networkmodule tem uns qualifier de exemplo, ai aqui vc faz um get(qualifier) e ja eera

Reply:
ai para funfar legal, o projeto ainda não tem, mas pode criar um modulo core-shared que vai ter que ter um modulo geral que todos consigam pegar nas dependencias para tu pegar essa instancia do qualifier. Mas se for muito trampo, assim ta de boas e depois a gente faz por parte

Reply:
boaa, criei o modulo `core-shared` e implementei o qualifier, quando puder veja se faz sentido o/